pr_2/javascript/random_samples/xxe_node.js

```
@@ -0,0 +1,71 @@
+    // SAX parser expands external entities
+    const parser = new libxmljs.SaxParser();
+    const x = 1
+    // ruleid:node_xxe
+    var products = parser.parseXmlString(req.files.products.data, { noent: true, noblanks: true })
+})
+
+// {/fact}
+
+// {fact rule=xml-external-entity@v1.0 defects=1}
+const express = require('express')
+const libxmljs = require('libxml')
```

Biome (2.1.2)

[error] Shouldn't redeclare 'libxmljs'. Consider to delete it or rename it.

'libxmljs' is defined here:


(lint/suspicious/noRedeclare)